src/test/resources/dokumentasjon/features/hovedregler/grunnforordningen/regel_2_eos_borger.feature

```diff
@@ -53,3 +53,34 @@ Egenskap: Eøs-land
       | GBR  | STORBRITANNIA | Ja   |
       | USA  | USA           | Nei  |
 
+
+  Scenario: Regel 2: Norsk statsborger med historisk statsborgerskap utenfor EØS får ja
+
+    Gitt følgende statsborgerskap i personhistorikken
+      | Landkode | Fra og med dato | Historisk |
+      | USA      | 30.01.2000      | Ja        |
```

Comment:
Hva vil det si at et statsborgerskap er historisk? Er det da utgått?

Reply:
Ja, det stemmer.

Reply:
Det er visst ganske god datakvalitet på det feltet, så vi kan lene oss på det for å avgjøre om statsborgerskap vi får fra pdl er gjeldende. Det samme flagget finnes for adresser også.